### PR TITLE
[Fix #2740] Bandage on the wound of WordArray autoconfig ridiculousness

### LINKED ISSUE #2740: --auto-gen-config sets a value to "-Infinity" for WordArray
I haven't investigated exactly why this is occurring yet, but running `rubocop --auto-gen-config` with Rubocop 0.36 on a codebase generated this config:

``` yaml
Style/WordArray:
  EnforcedStyle: percent
  MinSize: -Infinity
```

And this config is invalid and leads to errors later when it's loaded. I don't think our word arrays are particularly unusual -- we have 4 of them, ranging in length from 2 to 3 elements.

### PR DESCRIPTION
**Pull request by alexdowad**
@bbatsov, another bug fix for you, kind sir. Sorry it's a year late.

### CODE REVIEW

**Comment by backus** [comment]
What a great commit message

**Comment by bbatsov** [comment]
The new example is failing.

**Comment by alexdowad** [comment]
Should be better now... let's just see if the build passes.

**Comment by bbatsov** [comment]
Seems that the build is still broken.

**Comment by alexdowad** [comment]
What????

**Comment by alexdowad** [comment]
Egads! Style errors! OK, let me try again... (Sorry!)
